### PR TITLE
remove old prop from db update

### PR DESCRIPTION
**Pull request by annyhe**
This change aims to improve performance by reducing data sent from scoket.io.
Originally updated objects would be sent with {
key: { old: oldObj, new: newObj }
}
This change takes out the old object, so the updated object is sent with {
key: { new: newObj }
}

### CODE REVIEW

File: db/helpers/common.js [hunk 1]
```diff
@@ -143,7 +143,6 @@ function prepareToPublish(inst, changedKeys, ignoreAttributes) {
   for (let i = 0; i < changedKeys.length; i++) {
     if (!ignoreSet.has(changedKeys[i])) {
       return {
-        old: inst._previousDataValues,
         new: inst.get(),
```
Comment (shriramshankar) [inline db/helpers/common.js]: @iamigo @annyhe we will need the old values to do some filtering in the realtime module. Can we remove the old values in /realtime/socketIOEmitter.js at line 30, just before the object is emitted to the client ?

Comment (annyhe) [inline db/helpers/common.js]: @shriramshankar Do we have tests for /realtime/socketIOEmitter.js? I don't want to break any functionality by a simple change.

Comment (shriramshankar) [inline db/helpers/common.js]: all the tests for the realtime events are under the realtime folder under tests.

**Comment by annyhe** [comment]
There is no large differences in duration between this and master's ws.onmessage and Manager.ondata events. However in the new version the handleEvent seems to be shorter.

Attached are pictures of the timeline with changes (purple), and without changes (green). 
screen shot 2016-11-09 at 9 50 06 am
screen shot 2016-11-09 at 9 49 47 am

The Timelines are attached in the zip file for reference (they are verbose)
[Archive 3.zip](https://github.com/salesforce/refocus/files/581486/Archive.3.zip)

**Comment by annyhe** [comment]
@shriramshankar Please review and comment

**Comment by shriramshankar** [comment]
@annyhe  looks good.

File: realtime/socketIOEmitter.js [hunk 2]
```diff
@@ -27,7 +27,8 @@ module.exports = (io, key, mssgObj) => {
   }
   for (const nsp in io.nsps) {
     if (nsp && rtUtils.shouldIEmitThisObj(nsp, obj)) {
-      io.of(nsp).emit(key, JSON.stringify(mssgObj));
+      // obj contains only the newest values
+      io.of(nsp).emit(key, JSON.stringify({ key: obj }));
```
Comment (iamigo) [inline realtime/socketIOEmitter.js]: > the updated object is sent with {
> key: newObj
> }

Actually, in order to *not* break any lenses, we need to send
```
{
  key: { new: newObj }
}
```

**Comment by annyhe** [comment]
@iamigo Updated this PR with requested changes.

File: realtime/socketIOEmitter.js [hunk 3]
```diff
@@ -27,7 +27,14 @@ module.exports = (io, key, mssgObj) => {
...
-      io.of(nsp).emit(key, JSON.stringify(mssgObj));
+      // obj contains only the newest values
+      io.of(nsp).emit(key, JSON.stringify({
+        key: {
+          new: {
+            obj,
+          },
+        },
+      }));
     }
```
Comment (shriramshankar) [inline realtime/socketIOEmitter.js]: @annyhe Can you make sure that you can still see the socketio events come into the browser after these changes?

Comment (annyhe) [inline realtime/socketIOEmitter.js]: Yes, the changes are still coming in. Attached a picture for details.
screen shot 2016-11-13 at 4 07 59 pm

Comment (shriramshankar) [inline realtime/socketIOEmitter.js]: Looks good. Ready to merge once the conflicts are resolved.

File: realtime/socketIOEmitter.js [hunk 4]
```diff
@@ -27,7 +27,12 @@ module.exports = (io, key, mssgObj) => {
   }
   for (const nsp in io.nsps) {
     if (nsp && rtUtils.shouldIEmitThisObj(nsp, obj)) {
-      io.of(nsp).emit(key, JSON.stringify(mssgObj));
+      // obj contains only the newest values
+      io.of(nsp).emit(key, JSON.stringify({
+        key: {
+          new: obj,
```
Comment (iamigo) [inline realtime/socketIOEmitter.js]: I approved too soon. I'd like to see a test for this change (or at least a reason why it's impractical to write that test right now).

Comment (annyhe) [inline realtime/socketIOEmitter.js]: @iamigo @meynet enzyme has MIT license. https://www.npmjs.com/package/enzyme

**Comment by annyhe** [comment]
@shriramshankar @iamigo Added test

**Pull request merged**